lintly/parsers.py

```diff
@@ -64,13 +64,22 @@ def parse_violations(self, output):
             if not match:
                 continue
 
-            path = self._normalize_path(match.group('path'))
+            def get_from_match(key, not_found_value=None):
+                if key in match.groupdict():
+                    return match.group(key)
+                return not_found_value
+
+            path = self._normalize_path(get_from_match("path"))
+            line = int(get_from_match("line", 0))
+            column = int(get_from_match("column", 0))
```

Comment:
Should the fallback values be `1` or `0` ?

Reply:
`0` sounds fine; I don't think Lintly uses the column number for any logic (other than for pure reporting purposes), so I don't think it will matter or break anything.

(Also `1` makes me think an issue was *actually* found at column 1, so maybe let's not do that.)

Reply:
Yes this numbers are just for annotations. So `0` is appropriate here.